### PR TITLE
Update/change account from wheelmap_visitor to wheelmap_streetspotr

### PR DESCRIPTION
**Pull request by 1000miles**
This PR does following:
- changes the user from `User.wheelmap_visitor` to `User.wheelmap_streetspotr` for upcoming photo imports via the rake task `bundle exec rake streetspotr:import`
- sets rules for image processing dependent on the environment via `config.enable_processing`

Related Github issue: #622 

Since this is built on PR #599, please merge that one first.

### CODE REVIEW

File: lib/tasks/streetspotr.rake [hunk 1]
```diff
@@ -1,168 +1,172 @@
...
       if osm_id.blank?
-        unless poi
-          puts 'Skipped: Invalid Photo Line.'
+        unless previous_poi
+          puts "Skipped: osm_id blank and POI not found!"
+          @skipped[:provided_poi] += 1
           next
+        else
+          current_poi = previous_poi
+          find_or_initialize_provided_poi(current_poi, provider)
```
Comment (emig) [inline lib/tasks/streetspotr.rake]: in this context, we are sure that if osm_id is nil, we can assign the photo to the previous found poi?

File: config/initializers/carrierwave.rb [hunk 2]
```diff
@@ -47,15 +47,14 @@ def effectively_resize_to_limit(width, height, _format = 'jpg', quality = 70)
...
-    config.enable_processing = false
-  end
-end
 
-CarrierWave.configure do |config|
-  config.permissions = 0o644
-  #  config.directory_permissions = 0755
-  config.storage = :file
-end
+    if Rails.env.production? || Rails.env.staging?
```
Comment (emig) [inline config/initializers/carrierwave.rb]: This can be rewritten as: 

config.enable_processing =  Rails.env.production? || Rails.env.staging?

**Comment by 1000miles** [comment]
Thank you!!!! 🎈 🍰 🍨 :)

**Comment by 1000miles** [comment]
Deployed to production today with release [20170626103531](https://github.com/sozialhelden/wheelmap/releases/tag/production-20170626103531).

**Pull request merged**